PostHog/PostHogConfig.swift

```diff
@@ -33,6 +33,11 @@ import Foundation
     /// Determines the behavior for processing user profiles.
     @objc public var personProfiles: PostHogPersonProfiles = .identifiedOnly
 
+    /// The identifier of the App Group that should be used to store shared analytics data.
+    /// Posthog will try to get the physical location of the App Group’s shared container, otherwise fallback to the default location
```

Comment:
```suggestion
    /// PostHog will try to get the physical location of the App Group’s shared container, otherwise fallback to the default location
```